### ISSUE TITLE
Size and weight are different in overview and character sheet

### ISSUE THREAD

**Issue by l0b0** [elyukai/optolith-client#1603]
### Please confirm the following

- [X] I checked the [current issues](https://github.com/elyukai/optolith-client/issues) for duplicate problems.

### Describe the bug

With a size set to 195 and weight set to 102, the character sheet shows size 78 and weight 204. If some conversion has to happen, then the units should probably be displayed in both places.

### To Reproduce

1. Set character size and weight
2. Open the character sheet

### Expected behavior

The same numbers should show in the character sheet as in the overview.

### Operating System

Linux

### Version

1.5.1

**Comment by elyukai** [comment]
I assume your application is set to English? Because this is the only language where different units are used. The format in which data is saved is still the "main" format (kg, cm, …), while every pure display of those data is converted into the actual unit used throughout the English books (pounds, feet, …). I probably forgot to add the units there, I'm going to change that. But the underlying data units will always be from the metric system, and I do think it could also lead to quite a bit of confusion if the character sheet would not use imperial units at all. What do you think about that issue?

**Comment by l0b0** [comment]
(I'm a metric person myself, but) I think asking imperial unit users to enter values in metric is *very* confusing. Ideally the conversion would all happen in the background, so that the UI (including input fields) always shows the human-readable values (metric or imperial), while the save file contains the metric value.

**Comment by elyukai** [comment]
Yes, I know. But at the time I made that decision I wasn’t sure how I should implement it, which is why I try to add unit hints everywhere. Like, in the edit window for items, the English input label for weight should be `Weight (in kg)`.

I strongly agree that a uniform use of units is way better and the best approach. Since I need to rework the items UI for the V2 anyway, I’ll reconsider (and most likely also take care of) this topic then.

**Comment by elyukai** [comment]
> it would be amazing to have an option to switch between those in options, working with 2 is bit of a pain when you don't know conversions cuz living in europe

That's a good idea, I'll consider that! ✨
